esx_service/utils/auth_data.py

```diff
@@ -877,24 +879,73 @@ def __create_default_tenant(self):
             err = error_code_to_message[ErrorCode.TENANT_CREATE_FAILED].format(auth_data_const.DEFAULT_TENANT, error_msg)
             logging.warning(err)
             return err
+        error_msg = tenant.set_default_datastore(self.conn, auth_data_const.VM_DS_URL)
+        if error_msg:
+            err = error_code_to_message[ErrorCode.DS_DEFAULT_SET_FAILED].format(auth_data_const.DEFAULT_TENANT,
+                                                                                error_msg)
+            logging.warning(err)
+            return err
         return None
 
+    def get_all_ds_privileges_dict(self):
+        """Form a dictionary with all_ds privileges used in creating of default tenant"""
+        return  [{'datastore_url': auth_data_const.ALL_DS_URL,
+                  'allow_create': 1,
+                  'max_volume_size': 0,
+                  'usage_quota': 0}]
```

Comment:
Should; the ALL_DS and VM_DS dicts be kept as global vs. embedded in this function and the ones below? Being global may be better.